packages/riipen-ui-docs/src/modules/components/MarkdownPage.jsx

```diff
@@ -11,141 +18,164 @@ import Menu from "src/modules/components/Menu";
+            </ButtonIcon>
+            <Popover
+              anchorEl={anchorState}
+              closeOnScrolled={false}
+              fullWidth
+              isOpen={menuOpen}
+              lockScroll
+            >
+              <Menu />
+            </Popover>
+          </React.Fragment>
+        </div>
+      </AppBar>
+      <div className="page">
+        <div className="menu">
+          <Menu />
+        </div>
+        <div className="content">
+          {sections.map((section, index) => {
+            if (demos && constants.REGEX.DEMO.test(section)) {
+              let options;
+
+              try {
+                options = JSON.parse(`{${section}}`);
+              } catch (err) {
+                console.error(err);
+                return null;
+              }
+
+              const name = options.demo;
+
+              if (!demos || !demos[name]) {
+                throw new Error(
+                  [
+                    `Missing demo: ${name}. You can use one of the following:`,
+                    Object.keys(demos)
+                  ].join("\n")
                 );
               }
 
-              return <MarkdownElement key={index} text={section} />;
-            })}
-            <div className="footer">
-              <Divider />
-              <Footer />
-            </div>
-          </div>
-          <div className="toc">
-            <TOC sections={sections} />
+              return <Demo key={index} demo={demos[name]} options={options} />;
+            }
+
+            return <MarkdownElement key={index} text={section} />;
+          })}
+          <div className="footer">
+            <Divider />
+            <Footer />
           </div>
         </div>
-        <style jsx>{`
-          .page {
-            display: flex;
-            justify-content: space-between;
-            padding-top: 22px;
-            position: relative;
-            top: 50px;
-            width: 100%;
-          }
-          .page :global(*:target) {
-            padding-top: 50px;
-          }
-          .content {
-            padding: 0 20px;
-            overflow-x: auto;
-            width: 100%;
-          }
-          .menu {
-            padding: ${theme.spacing(2)}px;
-          }
+        <div className="toc">
+          <TOC sections={sections} />
+        </div>
+      </div>
+      <style jsx>{`
+        .page {
+          display: flex;
+          justify-content: space-between;
+          padding-top: 22px;
+          position: relative;
+          top: 50px;
+          width: 100%;
+        }
+        .page :global(*:target) {
+          padding-top: 50px;
+        }
+        .content {
+          padding: 0 20px;
+          overflow-x: auto;
+          width: 100%;
+        }
+        .menu {
+          padding: ${theme.spacing(2)}px;
+        }
+        .menuMobile {
+          display: none;
+        }
+        .menuMobile :global(.popover) {
+          overflow-y: auto;
```

Comment:
A bit of hackery to make the popover scrolle :/

Reply:
Can we pass `classes` to the popover? If not maybe we should add that ability instead of this.